lib/virtfs/ntfs/attrib_file_name.rb

```diff
@@ -47,7 +46,7 @@ class FileName
     NS_DOS    = 2
     NS_DOSWIN = 3
 
-    UNNAMED = '[unnamed]'.AsciiToUtf8.freeze
+    UNNAMED = NTFS::Utils.AsciiToUtf8('[unnamed]').freeze
```

Comment:
This is the only caller to AsciiToUtf8, and it's in a constant, so I don't think it's necessary.  UTF-8 is the default encoding, so this code is basically doing "assume this already UTF-8 string is not UTF-8 then recode it as UTF-8", which is pointless.

If this is removed, then we don't need the AsciiToUtf8 method at all.

```suggestion
    UNNAMED = '[unnamed]'.freeze
```

Reply:
Wondered about that.